endoreg_db/models/media/video/video_file_meta/get_fps.py

```
@@ -6,7 +6,7 @@
     from ..video_file import VideoFile
 
 logger = logging.getLogger(__name__)
-def _get_fps(video: "VideoFile") -> Optional[float]:
+def _get_fps(video: "VideoFile") -> float:
```

Pylint (3.3.7)

[refactor] Too many branches (18/12)

(R0912)